src/bootstrap/dist.rs

```diff
@@ -1890,6 +1890,14 @@ fn maybe_install_llvm(builder: &Builder<'_>, target: TargetSelection, dst_libdir
         }
     }
 
+    let copy = |src: &_, dst, perms| {
+        if builder.config.llvm_from_ci {
```

Comment:
Why does this matter? I would expect our treatment here to be the same for both sources of llvm. If the issue is that the LLVM build sometimes produces symlinks, and we never have those in the output (since our tarballs deref or whatever), then I think we should check for symlinks here rather than making an assumption here.

Reply:
Hmm, ok. Done, and checked that it still has no symlinks in the final tarball.